wagtail/admin/templates/wagtailadmin/shared/human_readable_date.html

```diff
@@ -1,6 +1,6 @@
 {% load wagtailadmin_tags i18n %}
 
-<button type="button" class="w-human-readable-date" data-tippy-content="{{ date|date:"DATETIME_FORMAT" }}" data-tippy-placement="{{ position }}">
+<button type="button" class="w-human-readable-date" data-controller="{{ date|date:"DATETIME_FORMAT" }}">
```

Comment:
Sorry, this is not correct, the attributes need to be as follows;


```suggestion
<button type="button" class="w-human-readable-date" data-controller="w-tooltip" data-w-tooltip-content-value="{{ date|date:"DATETIME_FORMAT" }}" data-w-tooltip-placement-value="{{ position }}>
```

Also, it looks like you accidentally removed the placement one from this file.